src/Microsoft.AspNetCore.Mvc.Core/Resources.resx

```diff
@@ -387,4 +387,7 @@
     <value>Unable to create an instance of type '{0}'. The type specified in {1} must not be abstract and must have a parameterless constructor.</value>
     <comment>0 is the type to configure. 1 is the name of the parameter, configurationType.</comment>
   </data>
+  <data name="ComplexTypeModelBinder_NoPrameterlessConstructor" xml:space="preserve">
+    <value>Could not create an instance of type '{0}' as it's either an abstract type or a struct or does not have a parameterless constructor.</value>
```

Comment:
Too much "or" and I'm not sure we concatenate much, if at all, in our resources. Suggest `Could not create an instance of type '{0}'. Model bound complex types must not be abstract or value types and must have a parameterless constructor.`

Since the other workaround is to set the property to a non-`null` value, could add something about that as well e.g. `Alternatively, set the `{1}` property to a non-null value in the `{2}` constructor.` I'm not positive this is necessary but it shouldn't hurt.